app/assets/stylesheets/govuk-component/_govspeak.scss

```diff
@@ -20,6 +20,10 @@
     text-align: start;
   }
 
+  strong {
```

Comment:
Talked about this IRL. I suggested that we should move both strong declerations into govspeak/typography and add a comment there that rich-govspeak is a… Govspeak ‘modifier’(?) than re-enables bold.